docs/docs/cloud/how-tos/human_in_the_loop_edit_state.md

```diff
@@ -0,0 +1,189 @@
+# How to Edit State of a Deployed Graph
+
+When creating LangGraph agents, it is often nice to add a human-in-the-loop component. This can be helpful when giving them access to tools. Often in these situations you may want to edit the graph state before continuing (for example, to edit what tool is being called, or how it is being called).
+
+This can be in several ways, but the primary supported way is to add an "interrupt" before a node is executed. This interrupts execution at that node. You can then use update_state to update the state, and then resume from that spot to continue.
+
+## Setup
+
+We are not going to show the full code for the graph we are hosting, but you can see it [here](https://langchain-ai.github.io/langgraph/how-tos/human_in_the_loop/edit-graph-state/#build-the-agent) if you want to. Once this graph is hosted, we are ready to invoke it and wait for user input. 
+
+### SDK initialization
+
+First, we need to setup our client so that we can communicate with our hosted graph:
+
+
+=== "Python"
+
+    ```python
+    from langgraph_sdk import get_client
+    client = get_client(url="whatever-your-deployment-url-is")
+    assistant_id = "agent"
+    thread = await client.threads.create()
+    ```
+
+=== "Javascript"
+
+    ```js
+    import { Client } from "@langchain/langgraph-sdk";
+
+    const client = new Client({apiUrl:"whatever-your-deployment-url-is"});
+    const assistantId = "agent";
+    const thread = await client.threads.create();
+    ```
+
+## Editing state
+
+### Initial invocation
+
+Now let's invoke our graph, making sure to interrupt before the `action` node.
+
+=== "Python"
+
+    ```python
+    input = {'messages':[{"role":"user","content":"search for weather in SF"}]}
+
+    async for chunk in client.runs.stream(
+        thread["thread_id"],
+        assistant_id,
+        input=input,
+        stream_mode="updates",
+        interrupt_before=["action"],
+    ):
+        if chunk.data and chunk.event != "metadata": 
+            print(chunk.data)
+    ```
+
+=== "Javascript"
+
+    ```js
+    const input = {
```

Comment:
let's make input look similar in both python and js so it's nicer to switch back and forth (i.e. if it's multiline for js, it should be also multiline for python)